spec/features/reviews/04_destroy_review_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+# As an authenticated user
+# I want to delete a review
+# So that I can remove it from a park's show page
+
+feature "review creator can edit review information" do
+
+  scenario "Review creator signs in and navigates to edit review page" do
+    review = create(:review)
+    park = review.park
+    user = review.user
+
```

Comment:
You're only using these here in one scenario, so not using `let` statements is fine!